src/service/operation-id-service.js

```diff
@@ -115,11 +115,14 @@ class OperationIdService {
     }
 
     async getCachedOperationIdData(operationId) {
-        if (this.memoryCachedHandlersData[operationId]) {
-            this.logger.debug(`Reading operation id: ${operationId} cached data from memory`);
-            return this.memoryCachedHandlersData[operationId].data;
+        try {
```

Comment:
i'm not sure if this try catch makes sense. Maybe just change the if condition to `this.memoryCachedHandlersData[operationId]?.data` ?

Reply:
Makes sense, i changed it